### DISCUSSION TITLE
Add copy and link action to interfaces and displays

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#14217] [Feature Requests]
I would love to see an option to add a _copy to clipboard_ as well as a _link_ action button to text interfaces and displays.

**This would allow to** 
1.) copy values
2.) Create a Link from the text input
booth in the collection overview as well as on the details page

If a text input is disabled there could also be an option to add "copy on click" that would copy the value once you click on that field.

**Comment by rijkvanzanten** [comment]
>  copy values

This also exists in the field context menu:

<img width="395" alt="CleanShot 2022-06-30 at 09 49 17@2x" src="https://user-images.githubusercontent.com/9141017/176694023-f115248c-6348-4450-b7d8-17723c6a34b3.png">


> Create a Link from the text input

Have you tried the button links interface? That's what we usually use for this use case 🙂

**Reply by Dominic-Marcelino** [reply]
### Copy Values:
That's true but what I'd  love to see is a pre-/appended icon to visualize it for non-technical users.

Next to that as mentioned the option to copy the value via click on a read-only field.

That makes things super convenient with IDs that you might need to copy on a regular base. Stripe for example does the same with userIDs, paymentIDs, etc in the dashboard.

I'd suggest the same for #12706

### Link
True, that's exactly what I've been looking for. Completely forgot that - thanks!
Maybe the button could also become configurable between a "link" and a "copy" button? 🤔

**Reply by rijkvanzanten** [reply]
> Maybe the button could also become configurable between a "link" and a "copy" button? 🤔

That'd be a great addition! 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
